ghost/member-attribution/lib/event-handler.js

```diff
@@ -0,0 +1,40 @@
+const {MemberCreatedEvent, SubscriptionCreatedEvent} = require('@tryghost/member-events');
+
+class MemberAttributionEventHandler {
```

Comment:
I don't think this should be part of the Member Attribution service, there's nothing here specific to attribution 🤔 

Long term, we should be emitting events from the members package, and then listening to them and storing them just like we are here.

I would have a `MemberEventService` somewhere which literally just listens to events and stores them, and then the attribution service is just used to convert history -> attribution.

That would keep all of the event storage logic in one place.

Having said that, I don't want to block this PR on it

Reply:
True, I think it would be nice to have all the event handlers in one place. Initially I did have a small dependency on the attribution package because originally the urlHistory was inside the events (not the attribution object).

Just to be sure, do you mean moving this code to the existing `members-events-service`, just like the `last-seen-at-updater`?

Reply:
Yeah I think the `members-events-service` would work, but I think we can create a generic "Event storage" or "Event persistence" handler, because it's not going to be running logic, it's going to be just a series of:
```js
DomainEvents.subscribe(Event, (event) => {
  Model.add(convertEventToModelData(event));
});
```

Reply:
Created a cleanup issue for this: https://github.com/TryGhost/Team/issues/1821